.stylelintrc.mjs

```diff
@@ -0,0 +1,21 @@
+/** @type {import('stylelint').Config} */
+export default {
+  extends: ['stylelint-config-standard'],
+  reportNeedlessDisables: true,
+  reportInvalidScopeDisables: true,
+  reportDescriptionlessDisables: true,
+  rules: {
+    'font-family-no-missing-generic-family-keyword': [
+      true,
+      {
+        ignoreFontFamilies: ['Font Awesome 5 Free'],
+      },
+    ],
+    'selector-class-pattern': null,
+    'selector-id-pattern': null,
+    // Fixable Stylelint 16 rules
+    'declaration-block-no-redundant-longhand-properties': null,
```

Comment:
Thinking further on this, it is better to ignore instead of what I was doing in #2853, as using `top right bottom left` is still so popularly used.